SourceX/qol.cpp

```diff
@@ -237,4 +237,156 @@ void AutoGoldPickup(int pnum)
 	}
 }
 
+struct drawingQueue {

```

Comment:
Let's put all the alt highlighting code into a separate file?

`drawingQueue` is a very general name making it very unclear what this code is about